modules/1_prepare/prepare.tf

```diff
@@ -154,7 +154,12 @@ resource "null_resource" "bastion_init" {
       "echo MTU=1450 | sudo tee -a /etc/sysconfig/network-scripts/ifcfg-$(ip r | grep ${ibm_pi_network.public_network.pi_cidr} | awk '{print $3}')",
       # Set specified mtu for private interface.
       "sudo ip link set dev $(ip r | grep \"${data.ibm_pi_network.network.cidr} dev\" | awk '{print $3}') mtu ${var.private_network_mtu}",
-      "echo MTU=${var.private_network_mtu} | sudo tee -a /etc/sysconfig/network-scripts/ifcfg-$(ip r | grep ${data.ibm_pi_network.network.cidr} | awk '{print $3}')"
+      "echo MTU=${var.private_network_mtu} | sudo tee -a /etc/sysconfig/network-scripts/ifcfg-$(ip r | grep ${data.ibm_pi_network.network.cidr} | awk '{print $3}')",
+      <<EOF
```

Comment:
Have you tested this change? I do not think this syntax is correct. Please see the job for the errors.

Reply:
Yes I have tested this code locally and its working fine, the job errors are for previous commit  in which below syntax error was present.

```
<<EOF     
     if [[ ${var.fips_compliant} = true ]]; then
	 fips-mode-setup --enable
     fi
EOF
```

I have attached screen shot for the for previous commit error 

<img width="541" alt="syntax_error_for_fips_bastion" src="https://user-images.githubusercontent.com/56910884/162772826-30fc52c8-acb9-46b4-8057-afcd25a6d06f.PNG">